spring/spring5/spring5-common/src/main/java/org/infinispan/spring/common/provider/PersistenceContextInitializer.java

```diff
@@ -0,0 +1,23 @@
+package org.infinispan.spring.common.provider;
+
+import org.infinispan.protostream.SerializationContextInitializer;
+import org.infinispan.protostream.annotations.AutoProtoSchemaBuilder;
+
+/**
+ * Interface used to initialise a {@link org.infinispan.protostream.SerializationContext} using the specified Pojos,
+ * Marshaller implementations and provided .proto schemas.
+ *
+ * @author Dan Berindei
+ * @since 12.1
+ */
+@AutoProtoSchemaBuilder(
+      includeClasses = {
+            NullValue.class,
+      },
+      schemaFileName = "persistence.spring.proto",
+      schemaFilePath = "proto/generated",
```

Comment:
maybe call it something with spring5 on it ?

Reply:
AFAICT the `schemaFilePath` is the same for all the context initializers.
Maybe you meant to comment on line 17?

Reply:
+1 to adding to `schemaFileName`

Reply:
@ryanemerson @karesti I didn't want to include spring5 because the Java packages don't include it either (e.g. `org.infinispan.spring.common.provider.NullValue`), but I can add it if you think it helps. WDYT about `persistence.spring5.proto`?

Reply:
Oops, forgot to make this change :)
I'll do it in the spring-session PR, since I will need to add a new SCI anyway.